src/index.tsx

```diff
@@ -134,6 +135,13 @@ function doPick<OS extends SupportedPlatforms>(
     throw new TypeError('Invalid copyTo option: ' + options.copyTo)
   }
 
+  if (
+    'initialPath' in options &&
+    (!!!options.initialPath || options?.initialPath?.length === 0)
```

Comment:
Also, it's probably best to add a `Platform.OS !== 'windows'` since this will fail there.

Alternatively, there is a more complex solution of checking for [allowed paths](https://learn.microsoft.com/en-us/uwp/api/windows.storage.pickers.pickerlocationid?view=winrt-22621) and converting them to the correct enum value. No idea why Windows handles it this way

Reply:
It's probably best to throw in a `Platform.OS !== 'windows'` as well, since this will fail on that platform.

Alternatively, there is a more complex solution of checking for allowed paths and converting them to the correct enum value, but even doing it that way will only work for the very first time it's opened, since Windows `filePicker` opens at the last used location regardless of `suggestedStartLocation`.

(p.s. sorry, I can't figure out how to add the code)